How to slient the ignored file warning

Because of my program using auto-generated .d.ts, which  is incompatible with my custom eslint rules, I had to ignore these files.

This issue is caused by `eslint 'ignore_file'` like:
```
❯ pnpm eslint "./src/auto-imports.d.ts"

/root/Programs/*****/auto-imports.d.ts
  0:0  warning  File ignored because of a matching ignore pattern. Use "--no-ignore" to override

✖ 1 problem (0 errors, 1 warning)
```


-------------
attachment 1:
emmm, is `eslint.dev.eslint`  defined in `vite.config.js`? THX

--- a/packages/vite-plugin-checker/src/checkers/eslint/main.ts
+++ b/packages/vite-plugin-checker/src/checkers/eslint/main.ts
@@ -81,8 +81,9 @@ const createDiagnostic: CreateDiagnostic<'eslint'> = (pluginConfig) => {
 
       const handleFileChange = async (filePath: string, type: 'change' | 'unlink') => {
         const absPath = path.resolve(root, filePath)
+        const isChangedFileIgnored = await eslint.isPathIgnored(filePath)
 
-        if (type === 'unlink') {
+        if (type === 'unlink' || isChangedFileIgnored) {
           manager.updateByFileId(absPath, [])
         } else if (type === 'change') {
           const diagnosticsOfChangedFile = await eslint.lintFiles(filePath)

--- a/packages/vite-plugin-checker/src/checkers/eslint/main.ts
+++ b/packages/vite-plugin-checker/src/checkers/eslint/main.ts
@@ -83,7 +83,9 @@ const createDiagnostic: CreateDiagnostic<'eslint'> = (pluginConfig) => {
         const absPath = path.resolve(root, filePath)
         const isChangedFileIgnored = await eslint.isPathIgnored(filePath)
 
-        if (type === 'unlink' || isChangedFileIgnored) {
+        if (isChangedFileIgnored) return
+
+        if (type === 'unlink') {
           manager.updateByFileId(absPath, [])
         } else if (type === 'change') {
           const diagnosticsOfChangedFile = await eslint.lintFiles(filePath)

